main/udev-init-scripts/APKBUILD

```diff
@@ -1,53 +1,33 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=udev-init-scripts
-pkgver=30
-pkgrel=6
+pkgver=32
+pkgrel=0
 pkgdesc="udev startup scripts for openrc"
 url="https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/sys-fs/udev-init-scripts/"
 arch="noarch"
 license="GPLv2"
-depends=""
-depends_dev=""
-makedepends="$depends_dev"
-replaces="udev"
+provides=udev
```

Comment:
Then this is actually a bug. Because provides=name will conflict with package named 'name' and this is the cycle after this:
udev (the wrapper package): depends=eudev
eudev: depends = udev-init-scripts
udev-init-scripts: provides udev

So you can no longer install the convenience package.

I think we should have eudev provides 'udev=$pkgver', remove the udev wrapper package, and remove the provides here.